client-api-studio/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "client-api-studio",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@openfin/workspace": "^1.8.0",
+        "@testing-library/jest-dom": "^5.14.1",
```

Comment:
Should some of these be dev dependencies? the testing library libs for example?

Reply:
Definitely yes, thank you!

Reply:
IMO, we should just remove any unused dependencies for now. These specifically seem unused:
 - `@testing-library/jest-dom`
 - `@testing-library/react`
 - `@testing-library/user-event`
 - `@types/jest`